components/widget/components/widgets-custom/carbon-flux/component.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import Emissions from 'assets/icons/widgets/carbon-flux/emissions.svg';
+import Removals from 'assets/icons/widgets/carbon-flux/removals.svg';
```

Comment:
I think we still need to update to the svg's with labels?

Reply:
@01painadam I can change these later if its needed

Reply:
it's resolved now, actually